corehq/apps/style/static/style/less/_hq/navs.less

```diff
@@ -1,3 +1,7 @@
+#hq-sidebar {
+  min-height: 1000px;
```

Comment:
Yeah, this is a temporary thing that's overridden inside either a `.ready` or `.load` callback (I forget which); so this is really only meant to take effect during the "split second" (which can actually be >1s) during which the page is being rendered through a variety of scripts and knockout bindings. The current behavior (without this) is that the sidebar colored area is very short and empty in the app builder for like a second and then expands when it's done. With this, it appears full height but empty for a second, and then gets filled in, which is less jittery/jarring.